prisma/seed.ts

```diff
@@ -66,6 +66,24 @@ const users = async () => {
         },
       },
     }),
+    prisma.emissionFactor.create({
```

Comment:
Pq crée un use case base empreinte ? qui en plus n'est pas lié à une version, ca fait un use case pas du tout reel alors qu'on a un vrai seed avec BE

Reply:
J'ai ajouté la versionId sur chaque FE créé. Mais c'est un ajout simple pour les tests e2e. Je ne comprends pas pourquoi il faudrait importer la vraie seed pour ces tests ?

Reply:
j'ai pas dis qu'il fallait importer la vrai seed, j'ai juste dit que la tu crées un FE de la base empreinte qui n'en est pas du tout un et que c'est un cas pas reel qui peut poser des problemes
alors que tu n'as pas du tout besoin qu'il vienne de la BE pour ton test

Reply:
Mais quelle est la différence avec les 2 autres FE de la seed ?

Reply:
ils sont pas taggués base empreinte

Reply:
C'est pas ça le tag base empreinte ?

![image](https://github.com/user-attachments/assets/6d929c00-4ea5-445d-b883-72f42b2acdc9)

J'ai littéralement copié/collé un FE valide et ai juste changé l'unité et les chiffres

Reply:
Effectivement, et ils ne devraient pas l'etre non plus :/